### PR TITLE
Fix merge conflicts

### PR DESCRIPTION
**Pull request by curstwist**
#### Summary
 ```SUMMARY: none```

#### Describe alternatives you've considered
leave until Sourslime has time to fix them, but I'm really looking forward to this PR's additions.

### CODE REVIEW

File: data/json/items/armor/jewelry.json [hunk 1]
```diff
@@ -1,4 +1,71 @@
...
+    "name": "friendship bracelet",
+    "description": "A homemade bracelet crafted from colourful knickknacks.  It's not very expensive, but it's the effort that makes it special.",
+    "looks_like": "copper_bracelet",
+    "weight": 20,
+    "volume": 0,
+    "price": 100,
+    "flags": [ "VARSIZE", "FANCY" ],
+    "material": [ "wood", "plastic" ],
+    "symbol": "[",
+    "color": "purple"
```
Comment (kevingranade) [inline data/json/items/armor/jewelry.json]: This color doesn't exist.

**Comment by SourSlime** [comment]
Thanks for doing this :)

**Comment by ZhilkinSerg** [comment]
I've messed a bit with this, but I will fix it tonight.

File: data/json/items/armor/jewelry.json (hunk 1 above)
Comment (ZhilkinSerg) [reply data/json/items/armor/jewelry.json]: I've changed it to `red`.

**Pull request merged**